src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonTableViewCell.swift

```diff
@@ -377,6 +390,16 @@ class HealthCertifiedPersonTableViewCell: UITableViewCell, ReuseIdentifierProvid
 				titleStackView.topAnchor.constraint(equalTo: gradientView.topAnchor, constant: 20.0),
 				titleStackView.trailingAnchor.constraint(equalTo: accessoryIconView.leadingAnchor, constant: 8.0),
 
+				maskStateView.leadingAnchor.constraint(equalTo: maskAdmissionStatesView.leadingAnchor),
+				maskStateView.topAnchor.constraint(equalTo: maskAdmissionStatesView.topAnchor),
+				maskStateView.bottomAnchor.constraint(equalTo: maskAdmissionStatesView.bottomAnchor),
+				maskStateView.widthAnchor.constraint(equalTo: maskAdmissionStatesView.widthAnchor, multiplier: 0.78),
+				
+				admissionStateView.trailingAnchor.constraint(equalTo: maskAdmissionStatesView.trailingAnchor),
+				admissionStateView.topAnchor.constraint(equalTo: maskAdmissionStatesView.topAnchor),
+				admissionStateView.bottomAnchor.constraint(equalTo: maskAdmissionStatesView.bottomAnchor),
+				admissionStateView.widthAnchor.constraint(equalTo: maskAdmissionStatesView.widthAnchor, multiplier: 0.20),
```

Comment:
Wouldn't it be better give each of these the space they need if only one of them needs to be present? Again thinking about potential longer admission states/mask states and larger font sizes

Reply:
Will be handled in https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13883